wire-swift-generator/src/main/java/com/squareup/wire/swift/SwiftGenerator.kt

```diff
@@ -158,6 +158,44 @@ class SwiftGenerator private constructor(
       else -> null
     }
 
+  private val Field.defaultedValue: CodeBlock?
+    get() {
+      when (val default = default) {
+        // identity values
+        null -> {
+          if (!isOptional) return null
+          if (type == ProtoType.ANY) return null
+          if (isMessage && !isRequiredParameter && !isCollection) {
+            val messageType = schema.getType(type!!) as MessageType
+            return if (messageType.fields.any { it.isRequiredParameter }) null else CodeBlock.of("%T()", messageType.typeName.makeNonOptional())
+          }
+          if (isEnum) {
+            val enumType = schema.getType(type!!) as EnumType
+            return if (enumType.constants.getOrNull(0) != null) CodeBlock.of("%T.%L", typeName.makeNonOptional(), enumType.constants[0].name) else null
+          }
+          when (typeName.makeNonOptional()) {
+            BOOL -> return CodeBlock.of("%L", false)
+            INT -> return CodeBlock.of("%L", 0)
+            INT32 -> return CodeBlock.of("%L", 0)
+            INT64 -> return CodeBlock.of("%L", 0)
+            UINT32 -> return CodeBlock.of("%L", 0)
+            UINT64 -> return CodeBlock.of("%L", 0)
+            FLOAT -> return CodeBlock.of("%L", 0)
+            DOUBLE -> return CodeBlock.of("%L", 0)
+            STRING -> return CodeBlock.of("%S", "")
+            DATA -> return CodeBlock.of(
+              "%T(base64Encoded: %S)!",
```

Comment:
Can you not just do `Data()` ?

Reply:
yeah it was an initial copy paste from below will be updating

Reply:
fixed